src/coreComponents/constitutive/contact/CoulombContact.hpp

```diff
@@ -232,47 +315,9 @@ void CoulombContactUpdates::computeTraction( localIndex const k,
 
     // Yield function (not necessary but makes it clearer)
     real64 const yield = tauNorm - limitTau;
-    if( yield < 0 )
-    {
-      // Elastic slip case
-
-      // Tangential components of the traction are equal to tau
-      tractionVector[1] = tau[0];
-      tractionVector[2] = tau[1];
-
-      dTractionVector_dJump[1][1] = m_shearStiffness;
-      dTractionVector_dJump[2][2] = m_shearStiffness;
-
-      // The slip is only elastic: we add the full slip to the elastic one
-      LvArray::tensorOps::add< 2 >( m_elasticSlip[k], slip );
-    }
-    else
-    {
-      // Plastic slip case
-
-      // Tangential components of the traction computed based on the limitTau
-      tractionVector[1] = limitTau * slip[0] / slipNorm;
-      tractionVector[2] = limitTau * slip[1] / slipNorm;
 
-      dTractionVector_dJump[1][0] = m_penaltyStiffness * dLimitTau_dNormalTraction * slip[0] / slipNorm;
-      dTractionVector_dJump[1][1] = limitTau * pow( slip[1], 2 )  / pow( LvArray::tensorOps::l2NormSquared< 2 >( slip ), 1.5 );
-      dTractionVector_dJump[1][2] = limitTau * slip[0] * slip[1] / pow( LvArray::tensorOps::l2NormSquared< 2 >( slip ), 1.5 );
-
-      dTractionVector_dJump[2][0] = m_penaltyStiffness * dLimitTau_dNormalTraction * slip[1] / slipNorm;
-      dTractionVector_dJump[2][1] = limitTau * slip[0] * slip[1] / pow( LvArray::tensorOps::l2NormSquared< 2 >( slip ), 1.5 );
-      dTractionVector_dJump[2][2] = limitTau * pow( slip[0], 2 )  / pow( LvArray::tensorOps::l2NormSquared< 2 >( slip ), 1.5 );
-
-      // Compute elastic component of the slip for this case
-      real64 const plasticSlip[2] = { tractionVector[1] / m_shearStiffness,
-                                      tractionVector[2] / m_shearStiffness };
-
-      LvArray::tensorOps::copy< 2 >( m_elasticSlip[k], slip );
-      LvArray::tensorOps::subtract< 2 >( m_elasticSlip[k], plasticSlip );
-    }
+    fractureState = yield < 0 ? FractureState::Stick : FractureState::Slip;
```

Comment:
Four different states are defined. When updating the FractureState, it seems that the `new_slip` state is missing?

Reply:
the efem solver does not distinguish between `new_slip` and `slip`.